.eslintrc.js

```diff
@@ -44,6 +44,7 @@ module.exports = {
     'jsdoc/require-returns': 'off',
     'jsdoc/no-types': 'warn',
     'prefer-arrow-callback': 'error',
+    'react/prop-types': 'off',
```

Comment:
Could you remove this and instead create the Type `pageOptions` property?

One of the bullet points for the refactor I want to do, is to stop using `any` and add types. It would be great if we don't add this configuration to the eslint.

Reply:
Thanks @hbcarlos
I agree but I'm not fully comfortable with this.
Should we add types for all the "pagination" options?
Should we recreate a new type for our use or use the ones from react?

Reply:
I created an interface for `IPaginationOptions` : https://github.com/mamba-org/quetz-frontend/pull/125/commits/762d5124b8636871c95a9fae92e2a379ac94d7d3

Reply:
Yes. It is perfect!